packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/quickLaunch.test.ts

```diff
@@ -29,10 +29,9 @@ import { TestDebuggerExecutor } from '../../../src/commands/quickLaunch';
 import { workspaceContext } from '../../../src/context';
 import { nls } from '../../../src/messages';
 
-const $$ = testSetup();
-
 // tslint:disable:no-unused-expression
 describe('Quick launch apex tests', () => {
+  const $$ = new TestContext();
```

Comment:
Declaring it outside the describe block, might interfere with int tests run after it. Saw the similar behavior in apex package. So keeping it within the describe block.